cmd/keeper/cmd/keeper.go

```diff
@@ -328,6 +343,11 @@ func (p *PostgresKeeper) createPGParameters(db *cluster.DB) common.Parameters {
 		parameters[k] = v
 	}
 
+	// Override parameters from those supplied to the keeper on boot
```

Comment:
We need that to be above the mandatory parameters: mandatory means mandatory!

Reply:
It looks like `wal_keep_segments` is in the `mandatoryPGParameters`, but we're going to want to override it.

That's a shame, cause `mandatory` is probably the wrong word for it! Let's keep this override where it is right now, but drop a comment that explains why we've placed it in a position where it could override 'mandatory' parameters.